Mathlib/CategoryTheory/Monoidal/Braided.lean

```diff
@@ -123,6 +128,30 @@ theorem braiding_naturality {X X' Y Y' : C} (f : X ⟶ Y) (g : X' ⟶ Y') :
   rw [tensorHom_def' f g, tensorHom_def g f]
   simp_rw [Category.assoc, braiding_naturality_left, braiding_naturality_right_assoc]
 
+theorem yang_baxter (X Y Z : C) :
```

Comment:
Is there some reasons why you put associators at the both ends in the RHS? I think this is one possible choice, but we also have other choices.

Reply:
Sorry, I meant LHS, not RHS.

Reply:
I chose this way of writing it because it becomes an equality of maps `X ⊗ Y ⊗ Z ⟶ Z ⊗ Y ⊗ X`, both the domain and codomain being fully right associated. Also because it's what was easiest for the proof of associativity in `mopBraidedFunctor`. There are other choices we could make but I don't see a reason to prefer another